packages/web-api/src/instrument.ts

```diff
@@ -11,7 +12,7 @@ function replaceSlashes(s: string): string {
 }
 
 const baseUserAgent = `${replaceSlashes(packageJson.name)}/${packageJson.version} ` +
-                      `node/${process.version.replace('v', '')} ` +
+                      `${basename(process.title)}/${process.version.replace('v', '')} ` +
                       `${os.platform()}/${os.release()}`;
```

Comment:
The improvement on the title part looks great to me! Can we do anything for the OS info part, which is currently `browser/undefined` with Deno?

Reply:
I _believe_ `browser/undefined` comes from the [`os-browserify` NPM package](https://www.npmjs.com/package/os-browserify) - and of course, we can shim in whatever values we want! I can look into writing our own miniature `os` compatibility shim if the beta deno one is insufficient or not ready or the Deno OS APIs are not implemented.

Reply:
Just checked - we could use the [built-in compatibility layer for the `os` node module from deno](https://deno.land/std@0.116.0/node/os.ts) but we require to use the latest deno (v1.16.2) and we must use with the `--unstable` flag. If we are ok with increasing the deno requirements in this way, then this can be tweaked to use the same approach as the [`deno-shims/qs-shim.js` file](https://github.com/slackapi/node-slack-sdk/blob/main/packages/web-api/deno-shims/qs-shim.js), something as easy as:

```
export * from 'https://deno.land/std@0.116.0/node/os.ts'
```

Reply:
Thanks for checking this! In the near future, our main users will use the latest Deno version for sure. If we have a reason to keep using a specific stable version, I think we don't need to rush for having the os module. In the case, we can have a TODO comment describing the necessity to update the part once the os module is available without unstable flag.

Reply:
@seratch I've added a TODO/comment explaining the situation, thanks for the suggestion! I will now resolve this conversation.